CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to MarkLogic-docker
+
+Thank you for your interest in contributing to this project! We welcome contributions from the community to make this project better.
+
+- [Found an Issue](#found-an-issue)
+- [Want a Feature](#want-a-feature)
+- [Getting Started](#getting-started)
+- [Issue Tracker](#issue-tracker)
+- [PR management](#pr-management)
+- [Code Style](#code-style)
+
+## Found an Issue?
+
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue 
+to our [GitHub Issue Tracker][Issue Tracker]. If you'd like to submit a feature enhancement, please first create an 
+issue with your proposed idea so that we can start a discussion about the problem you want to solve and what the best
+solution would be.  
+
+## Want a Feature?
+
+You can request a new feature by submitting an issue to our [GitHub Issue Tracker][Issue Tracker].  If you
+would like to implement a new feature then first create a new issue and discuss it with one of our
+project maintainers.
+
+## Getting Started
+
+To get started with contributing, please follow these steps:
+
+1. Fork the repository and clone it to your local machine.
+2. Install the necessary dependencies.
+3. Create a new branch for your changes.
+4. Make your desired changes to the codebase.
+5. Test your changes thoroughly.
+6. Tests can be done using the test framework. See [README](./test/README.md)
+
+## Issue Tracker
+
+If you encounter any issues or have suggestions for improvements, please open an issue in the issue tracker. Be sure to provide as much detail as possible, including steps to reproduce the issue if applicable.
+
+## PR management
+
+Created PR will not be merge as is.
+The MarkLogic docker team will use the PRs for "inspiration" but not merge the changes in directly. They may rewrite the code as they like, incorporating the submitted changes into their own code.
+
+**Important:** Please open an issue in the [Issue Tracker][] and get your proposed changes pre-approved by at least one of the project maintainers before you start coding. Nothing is more frustrating than seeing your hard work go to waste because your vision does not align with that of the project maintainers.
+
+## Code Style
+
+Please follow the established code style guidelines when making changes to the codebase. This ensures consistency and makes it easier for others to review and maintain the code.
```

Comment:
I don't think this section adds any value here unless we have specific guidelines to follow.